sklearn/utils/metaestimators.py

```diff
@@ -10,11 +10,30 @@
 __all__ = ['if_delegate_has_method']
 
 
+def _hasattr_nested(obj, attr):
```

Comment:
hm I would have thought operator.attrgetter  would solve this, but I guess it doesn't. That's kinda weird.